src/index.html

```diff
@@ -22,6 +22,355 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
```

Comment:
All navigation links point to 'index.html'. If these links are intended to navigate to different pages, ensure the 'href' attributes are updated accordingly.